react-toggle-switch-component/src/toggle-switch.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+class ToggleSwitch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { status: false };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.setState({ status: !this.state.status });
+  }
+
+  offOn() {
+    if (this.state.status === false) {
+      return 'on';
+    } else {
+      return 'off';
+    }
+  }
+
+  buttonText() {
+    if (this.state.status === false) {
+      return 'on';
+    } else {
+      return 'off';
+    }
+  }
```

Comment:
Since these do the exact same thing you can just have one method and use it for the className and the text